winlogbeat/config/config.go

```diff
@@ -36,7 +36,7 @@ type Settings struct {
 // Validate validates the Settings data and returns an error describing
 // all problems or nil if there are none.
 func (s Settings) Validate() error {
-	validKeys := []string{"winlogbeat", "output", "shipper", "logging"}
+	validKeys := []string{"winlogbeat", "output", "beat", "logging"}
```

Comment:
For this strict validation code I would leave `shipper` in place until it is not longer supported. Right now both shipper and beat can be used.